popSimManu.tex

```diff
@@ -428,6 +377,70 @@ \subsection*{The Species Catalog}
+Currently, \stdpopsim uses the  \texttt{msprime} coalescent simulator \citep{kelleher2016efficient}
+as the default simulation engine.
+
+Coalescent simulations, while highly efficient, are limited in their ability to model
+complex selection scenarios, such as recurrent sweeps and background selection. For these reasons
+we have also implemented the forward-time simulator, \texttt{SLiM} \citep{haller2019tree,haller2019slim}, 
+as an alternative backend engine to \texttt{stdpopsim}, allowing for the simulation of processes that cannot be modeled under 
+the coalescent. However, as forward-time simulators explicitly model all individuals in a population, simulating large
+population sizes can be highly demanding of computational resources. One common practice used to address this challenge is to
+scale down the population size used in the simulations (\textit{N}),
+and then scale down times specified in the demographic model by the same factor, and scale up the mutation and recombination
+rates and selection coefficients such that the scaled parameters, \textit{N$\mu$}, \textit{Nr},
+and \textit{Ns}, remain unchanged \citep[see the \texttt{SLiM} manual and][]{uricchio2014robust}.
+Such down-scaled simulations are not completely equivalent to simulating 
+all individuals in the population, and may lead to subtle differences.
+However, since many sequence-based measures of population diversity remain unchanged when rescaling in this fashion,
+this practice is very common when using forward-time simulators, such as \texttt{SLiM}.
+
+We validated our implementation of the \texttt{SLiM} engine by comparing
+estimates of several population genetic summary statistics for neutral simulations generated by both \texttt{SLiM}
+and \texttt{msprime} under different down-scaling factors
+(Figures \ref{fig:slim_val_map} and \ref{fig:slim_val_nomap}). This comparison indicates that scaling down the simulated
+population size by up to a factor
+of $50$ ($Q=50$ in \texttt{SLiM}) produces patterns of both diversity and linkage disequilibrium that are
+qualitatively similar to those observed under the coalescent (i.e. \texttt{msprime}), when both are simulated under a recombination map
+(Figure \ref{fig:slim_val_map}).
+We do observe a slight departure in values of Tajima's D when scaling down by a factor of $50$,
+likely due to the subtle differences implied by down-scaling.
+Interestingly, this effect of down-scaling is more prominent and extends to other sequence-based statistcs
+when data sets are simulated under a uniform recombination rate (Figure \ref{fig:slim_val_nomap}).
+Importantly, both comparisons validate the equivalence of \texttt{SLiM} and \texttt{msprime} when no down-scaling is applied,
+and they suggest that down-scaling population sizes can significantly reduce running time at a fairly modest cost to accuracy. 
+However, since the effects of rescaling are not fully understood, then relatively small scale factors should be used.
```

Comment:
Relative to what? The time span of the simulation? Ne? If Ne, then which population?

Reply:
I agree that the language I used was vague. I like your edit here.